test/test_mock_iq_backend.py

```diff
@@ -70,6 +72,50 @@ def compute_probabilities(self, circuits: List[QuantumCircuit]) -> List[Dict[str
         return output_dict_list
 
 
+class MockIQReadoutAmplitudeHelper(MockIQExperimentHelper):
+    """functions needed for readout-amplitude experiment on mock IQ Backend"""
+
+    def __init__(
+        self,
+        alter_centers=True,
+        alter_widths=True,
+    ) -> None:
+        """Construct a MockIQReadoutAmplitudeHelper instance.
+
+        Args:
+            alter_centers (bool, optional): Whether to alter the centers of the IQ clusters based on the circuit 'xval'. Defaults to True.
+            alter_widths (bool, optional): Whether to alter the widths of the IQ clusters based on the circuit 'xval'. Defaults to True.
```

Comment:
Minor, but we _tend_ to put the type hints into the function signature.
```suggestion
            alter_centers: Whether to alter the centers of the IQ clusters based on the circuit 'xval'. Defaults to True.
            alter_widths: Whether to alter the widths of the IQ clusters based on the circuit 'xval'. Defaults to True.
```

Reply:
Addressed in 564f227.